packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -235,6 +235,16 @@ class TextEditingController extends ValueNotifier<TextEditingValue> {
 /// is a full-featured, material-design text input field with placeholder text,
 /// labels, and [Form] integration.
 ///
+/// ## Gesture Events Handling
+///
+/// This widget will provide rudimentary, platform-agnostic gesture event
```

Comment:
will provide => provides

I think "gesture handling" is the same as "gesture event handling"